products/jbrowse-web/src/rootModel.ts

```diff
@@ -88,7 +88,6 @@ export default function RootModel(
       ),
       savedSessionsVolatile: observable.map({}),
       textSearchManager: new TextSearchManager(pluginManager),
-      pluginManager,
```

Comment:
rider change, but this looked unused, and probably better to not have it unnecessarily. pluginmanager can/should be obtained from getEnv